apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.js

```diff
@@ -91,6 +92,10 @@ router.post('/', function (req, res, next) {
             },
             (err, result) => {
               if (ERR(err, next)) return;
+              flash(
+                'success',
+                'Successfully copied assessment. You are now viewing your copy of the assessment.',
```

Comment:
For consistency with other messages:

```suggestion
                'Assessment copied successfully. You are now viewing your copy of the assessment.',
```

Reply:
Resolved in commit e8d831c.

Reply:
I don't really like "your copy" here...

Reply:
What would you prefer? "the new copy"? Just "the copy"?

Reply:
The new copy is fine. Your copy is weird in this context, since you're likely copying from a context where the user also has access to, so the old could also be "yours".

Reply:
Since this is just matching the pattern established on other pages, I'm going to merge this as-is. Feel free to open a followup PR with different language if desired.